sw/device/lib/aes.h

```diff
@@ -0,0 +1,95 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef _F_LIB_AES_H__
+#define _F_LIB_AES_H__
+
+#include <stddef.h>
+#include <stdint.h>
+
+/**
+ * Supported AES modes
+ */
+typedef enum aes_modes { AES_ENC = 0, AES_DEC = 1 } aes_modes_t;
+
+/**
+ * AES unit configuration options.
+ */
+typedef struct hmac_cfg {
+  /** Operational mode @see aes_modes. */
+  aes_modes_t mode;
+  /** Can be 16, 24 or 32. */
+  size_t key_len_in_bytes;
+  /** Set to 1 to only start upon getting a trigger signal. */
+  unsigned char manual_start_trigger;
```

Comment:
This looks like it wants to be a `bool`.

Reply:
Done.